cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1668,26 +1668,8 @@ SwiftGenerator::MetadataVisitor::visitModuleStart(const ModulePtr& p)
         // top-level module
         const UnitPtr unit = p->unit();
 
```

Comment:
This logic was just a duplicate of the `getSwiftModule` function that you can see above here.